app/database/migrations/20200208123841_skilled_impact_requests.js

```diff
@@ -5,7 +5,8 @@ exports.up = function (knex) {
       .references('campaigns.camp_id')
       .onDelete('CASCADE')
       .onUpdate('CASCADE');
-    table.enum('skill', null, { useNative: true, enumName: 'enum_skills', existingType: true }).notNullable();
+    table.enum('skill', null, { useNative: true, enumName: 'enum_skills', existingType: true })
+      .notNullable();
```

Comment:
**nit**: Nothing was really changed here but can you please revert? Nice to keep the git history clean on migrations.